config/kubernetes_config.yaml

```diff
@@ -0,0 +1,27 @@
+cerberus:
+    distribution: kubernetes                             # Distribution can be kubernetes or openshift
+    kubeconfig_path: ~/.kube/config                      # Path to kubeconfig
+    watch_nodes: True                                    # Set to True for the cerberus to monitor the cluster nodes
+    watch_cluster_operators: False                       # Set to True for cerberus to monitor cluster operators. Enable it only when distribution is openshift
+    watch_namespaces:                                    # List of namespaces to be monitored
+        -    kube-system                                 # When enabled, it will check for the cluster sdn and monitor that namespace
```

Comment:
We need to remove the comment "When enabled, it will check for the cluster sdn and monitor that namespace".